monitoring/iris/views.py

```diff
@@ -0,0 +1,36 @@
+from django.shortcuts import render
+from datetime import datetime, timedelta
+
+from django.db.models import Max
+from django.db.models.functions import Lower
+
+from rest_framework import viewsets
+from rest_framework.renderers import TemplateHTMLRenderer
+
+
+from monitoring.iris.models import IrisCloudAndGrid
+
+from monitoring.iris.serializers import IrisCloudAndGridSerializer
+
+class IrisViewSet(viewsets.ReadOnlyModelViewSet):
+    # Lower('SiteName'): sorts sites alphabetically, case-insensitively.
+    # '-UpdateTime': sorts records within each site by UpdateTime in descending order (latest first).
+    queryset = IrisCloudAndGrid.objects.all().order_by(Lower('SiteName'), '-UpdateTime')
```

Comment:
I think I'd rather go for a standard grid then cloud ordering (reverse source type) within sites, otherwise the order will shift.